CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
-#  Change Log
```

Comment:
rethinking about this, it may actually conflict with the auto-generation of this file. Maybe we can only include `Examples.md` file for prettier scripts?

Reply:
makes sense 👍